### PR TITLE
GUAC-84 Leaderboard

### PR DESCRIPTION
**Pull request by WeezMatt**
## 61 ~GUAC-78-Tag-Logo-NameFix~ UPDATE PLUS LEADERBOARD STARTED

- edited the gif file to animate slower. The slow framerate is less distracting and matches the pixel theme.
- tab icon and image changed.
- fixed the hard code on the about page for the accordion feature. turned into a hook for reusability.
   (the component was added for the accordion itself and is no longer reliant on the team data, which is stored in a hook for 
   reference and to keep about page compact)
- altered index page button sizes added leaderboard button.
- added boxes for achievement and leaderboards with hover effect for future linking.
- pages now will update with the name of the page that you are on on the web tab 
- leaderboard page added with mock data until point system is created and connection to backend is made
- color update to buttons to match discussed style
- fixed issue where on smaller screens boxes covered nav buttons.

### CODE REVIEW

File: application/frontend/src/routes/leaderboard.tsx
Comment (mosguinz) [inline application/frontend/src/routes/leaderboard.tsx:1]: Good shit.

**Review by mosguinz — APPROVED** [review]
Great work. Thanks @WeezMatt!

**Review by yan-chan — APPROVED** [review]
Thank you! Good changes and I like the leaderboard :). The refactoring helps a lot too so thank you for doing that. Ran fine on my desktop as well and nothing broke *locally at least*

**Comment by WeezMatt** [comment]
Thank you! Should be good to merge now I believe!

**Pull request merged**